pkgs/development/python-modules/playwright/default.nix

```diff
@@ -93,10 +94,14 @@ buildPythonPackage rec {
 
   passthru = {
     inherit driver;
-    tests = {
-      driver = playwright-driver;
-      browsers = playwright-driver.browsers;
-    };
+    tests =
+      {
+        driver = playwright-driver;
+        browsers = playwright-driver.browsers;
+      }
+      // lib.optionalAttrs stdenv.isLinux {
```

Comment:
We could make this unconditional but whatever. Fine by me.

Reply:
I think it won't hurt to be explicit about supported platforms. :)